### PR TITLE
🐛 Fixking up some bugs with links

### PR DESCRIPTION
**Pull request by P-Saha**
**Description of changes**

**Type of Change**

- [x] Bug
- [ ] Styling
- [ ] New Feature

**Checklist before requesting review:**

- [ ] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [ ] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [x] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

File: pages/maintenance.tsx [hunk 1]
```diff
@@ -49,8 +50,11 @@ export default createPage({
             We'll be back up and running as quickly as possible. We appreciate your patience.
           </Typography>
           <Typography variant="subtitle2">
-            If you have any questions, <Link href={CONTACT_PAGE_PATH}>Contact Us</Link> or head back{' '}
-            <Link href="/">Home</Link>.
+            If you have any questions, <HyperLink href={CONTACT_PAGE_PATH}>Contact Us</HyperLink> or
```
Comment (hlminh2000) [inline pages/maintenance.tsx]: `CONTACT_PAGE_PATH` is internal, shouldn't this need `next/link` too

File: uikit/Footer/index.tsx [hunk 2]
```diff
@@ -59,7 +59,9 @@ const Footer = ({
             if (idx !== links.length - 1) {
               return (
                 <React.Fragment key={displayName}>
-                  <A href={href}>{displayName}</A>
+                  <A target={displayName === 'Documentation' ? '_blank' : '_self'} href={href}>
```
Comment (hlminh2000) [inline uikit/Footer/index.tsx]: let's not use contents to infer functionality, instead we can extend the `links` to include to include an optional `target` field in its content

File: pages/maintenance.tsx (hunk 1 above)
Comment (P-Saha) [reply pages/maintenance.tsx]: Resolved!

File: uikit/Footer/index.tsx (hunk 2 above)
Comment (P-Saha) [reply uikit/Footer/index.tsx]: Resolved!

**Pull request merged**